### PR TITLE
#265: update contributing.md

### PR DESCRIPTION
**Pull request by alexisbarnes**
Completes Issue #265. Did I miss anything @mcelestini?

### CODE REVIEW

File: src/components/Committee.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import React, { useEffect, useCallback } from 'react'
```
Comment (mcelestini) [inline src/components/Committee.js]: did we mean to add this committee component and hook? I don't see any return here so I don't think this component will render

**Review by SchaeStewart — APPROVED** [review]

**Review by mcelestini — APPROVED** [review]

**Comment by mcelestini** [comment]
very nice 💯 👏

File: src/components/Committee.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+import { useParams } from 'react-router-dom'
+import { Grid, GridContainer } from '@trussworks/react-uswds'
```
[the remark this answers is not in the document]
Comment (alexisbarnes) [reply src/components/Committee.js]: Hmmm...not sure. I'll see if I can fix it later.

File: src/components/Committee.js (hunk 1 above)
Comment (alexisbarnes) [reply src/components/Committee.js]: This shouldn't be in this PR. I'll fix in a bit.

Comment (mcelestini) [reply src/components/Committee.js]: looks good now with whatever changes you pulled in 👍

**Review by mcelestini — APPROVED** [review]
approving after most recent changes 👍

**Pull request merged**